packages/babel-types/src/definitions/core.ts

```diff
@@ -1625,17 +1662,14 @@ defineType("ExportNamedDeclaration", {
         assertEach(assertNodeType("ImportAttribute")),
       ),
     },
-    ...(process.env.BABEL_TYPES_8_BREAKING
-      ? {}
-      : {
-          assertions: {
-            optional: true,
-            validate: chain(
-              assertValueType("array"),
-              assertEach(assertNodeType("ImportAttribute")),
-            ),
-          },
-        }),
+    assertions: {
+      deprecated: true,
+      optional: true,
+      validate: chain(
+        assertValueType("array"),
+        assertEach(assertNodeType("ImportAttribute")),
+      ),
+    },
```

Comment:
I reverted the changes from a previous PR to this and added `deprecated`, because otherwise `any` would break our type system.